toke.c

```diff
@@ -2144,7 +2144,15 @@ S_check_uni(pTHX)
  * with a subroutine, S_lop() for which LOP is just another name.
  */
 
-#define LOP(f,x) return lop(f,x,s)
+#define LOP(f,x) return lop(LSTOP,f,x,s)
+
+/*
+ * BLKLOP : a macro to build a list operator. Similar to LOP but its first
+ * argument must be a block, so it does not need the PL_expect value to be
+ * specified.
+ */
```

Comment:
I forget to add `BLKLSTOP` to `debug_tokens` (in `toke.c`) in my proof of concept patch and apparently you did too.